app/api/v2/common/oauth.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import annotations
```

Comment:
This import is breaking FastApi's `depends`. Related issue: https://github.com/tiangolo/fastapi/issues/1654

Reply:
yeah i ran into this a long while ago at work, nice work actually tracking it down lol, that's not an easy error